recipe-client-addon/lib/ClientEnvironment.jsm

```diff
@@ -5,21 +5,20 @@
 "use strict";
 
 const {classes: Cc, interfaces: Ci, utils: Cu} = Components;
-Cu.import("resource:///modules/ShellService.jsm");
-Cu.import("resource://gre/modules/AddonManager.jsm");
 Cu.import("resource://gre/modules/Preferences.jsm");
-Cu.import("resource://gre/modules/Services.jsm");
 Cu.import("resource://gre/modules/Task.jsm");
-Cu.import("resource://gre/modules/TelemetryArchive.jsm");
 Cu.import("resource://gre/modules/XPCOMUtils.jsm");
-Cu.import("resource://shield-recipe-client/lib/NormandyApi.jsm");
+
+XPCOMUtils.defineLazyModuleGetter(this, "Services", "resource:///modules/Services.jsm");
```

Comment:
Gah, sorry, I missed that you had Services.jsm as well - you can Cu.import that as it'll generally be loaded anyway.